dev_requirements.txt

```diff
@@ -1,7 +1,3 @@
-# install latest changes in dbt-core
-#git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-#git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
```

Comment:
Is there a reason why this is getting removed?

Reply:
Those install the latest from github, not the latest release.